build/Dockerfile.base

```diff
@@ -30,69 +30,70 @@ ARG DEBIAN_FRONTEND="noninteractive"
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 # hadolint ignore=DL3003,DL3008,DL3013,SC1091
 RUN groupadd --gid 53150 -r easy_infra \
- && useradd -r -g easy_infra -s "$(which bash)" --create-home --uid 53150 easy_infra \
```

Comment:
This indentation is so that the commands align vertically; it looks like your linter may have changed it so there's always 3 spaces at the beginning.  Can you point me to the autoreformat tool that did this?  Either I want to align with yours or vice versa; not a fan of linter fights like this.